Packs/FeedOpenCTI/Integrations/FeedOpenCTI_v2/FeedOpenCTI_v2.py

```diff
@@ -0,0 +1,710 @@
+    """ Delete indicator from opencti
+
+        Args:
+            client: OpenCTI Client object
+            args: demisto.args()
+
+        Returns:
+            readable_output, raw_response
+        """
+    indicator_id = args.get("id")
+    try:
+        client.stix_cyber_observable.delete(id=indicator_id)
+    except Exception as e:
+        demisto.error(str(e))
+        raise DemistoException("Can't delete indicator.")
+    return CommandResults(readable_output='Indicator deleted.')
+
+
+def indicator_field_update_command(client: OpenCTIApiClient, args: dict) -> CommandResults:
+    """ Update indicator field at opencti
+
+        Args:
+            client: OpenCTI Client object
+            args: demisto.args()
+
+        Returns:
+            readable_output, raw_response
+        """
+    indicator_id = args.get("id")
+    # works only with score and description
+    key = KEY_TO_CTI_NAME[args.get("field")]  # type: ignore
+    value = args.get("value")
+    try:
+        result = client.stix_cyber_observable.update_field(id=indicator_id, key=key, value=value)
+    except Exception as e:
+        demisto.error(str(e))
+        raise DemistoException("Can't update indicator.")
+
+    return CommandResults(
+        outputs_prefix='OpenCTI.Indicator',
+        outputs_key_field='id',
+        outputs={'id': result.get('id')},
+        readable_output=f'Indicator {indicator_id} updated successfully.',
+        raw_response=result
+    )
+
+
+def indicator_create_command(client: OpenCTIApiClient, args: Dict[str, str]) -> CommandResults:
+    """ Create indicator at opencti
+
+        Args:
+            client: OpenCTI Client object
+            args: demisto.args()
+
+        Returns:
+            readable_output, raw_response
+        """
+    indicator_type = args.get("type")
+    created_by = args.get("created_by")
+    marking_id = args.get("marking_id")
+    label_id = args.get("label_id")
+    external_references_id = args.get("external_references_id")
+    description = args.get("description")
+    score = arg_to_number(args.get("score", '50'))
+    data = {}
+    try:
+        data = json.loads(args.get("data")) if args.get("data") else {}  # type: ignore
+    except Exception:
+        raise DemistoException("Data argument type should be json")
+
+    data['type'] = XSOHR_TYPES_TO_OPENCTI.get(indicator_type.lower(), indicator_type)  # type: ignore
+    simple_observable_key = None
+    simple_observable_value = None
+    if 'file' in indicator_type.lower():  # type: ignore
+        simple_observable_key = FILE_TYPES.get(indicator_type.lower(), indicator_type)  # type: ignore
+        simple_observable_value = data.get('hash')
+        if not simple_observable_value:
+            raise DemistoException("Missing argument in data: hash")
+    try:
+        # cti code prints to stdout so we need to catch it.
+        old_stdout = sys.stdout
+        sys.stdout = StringIO()
```

Comment:
Why do we have it only in this function?

Reply:
because only client.stix_cyber_observable.create function prints to stdout.